go/enclave/enclave_admin_service.go

```diff
@@ -600,3 +601,10 @@ func exportCrossChainData(ctx context.Context, storage storage.Storage, fromSeqN
 	} // todo: check fromSeqNo
 	return bundle, nil
 }
+
+func getSignatureValidator(useInMemDB bool, storage storage.Storage) (components.SequencerSignatureVerifier, error) {
+	if useInMemDB {
```

Comment:
Hopefully we have some checks in place that would barf for this but if we're in proper, SGX, attesting mode we definitely shouldn't be able to set that flag to true. In prod we'll lock it down with the enclave.json